internal/service/waf/flex.go

```diff
@@ -17,7 +17,7 @@ func expandAction(l []interface{}) *waf.WafAction {
 	}
 }
 
-func expandOverrideAction(l []interface{}) *waf.WafOverrideAction {
+func ExpandOverrideAction(l []interface{}) *waf.WafOverrideAction {
```

Comment:
```suggestion
func expandOverrideAction(l []interface{}) *waf.WafOverrideAction {
```

It's only used in `waf`.